--- zgv-5.7/src/bdf2h.c.orig	2000-06-01 16:44:48.000000000 +0200
+++ zgv-5.7/src/bdf2h.c	2003-08-02 20:54:36.000000000 +0200
@@ -46,12 +46,12 @@
 printf("#ifndef DEFINED_STRUCT_FONTINFO_TAG\n");
 printf("#define DEFINED_STRUCT_FONTINFO_TAG 1\n");
 printf("\
-struct fontinfo_tag
-  {
-  signed char *data;
-  int table[96];
-  int yofs,fh,oy;
-  };
+struct fontinfo_tag\n\
+  {\n\
+  signed char *data;\n\
+  int table[96];\n\
+  int yofs,fh,oy;\n\
+  };\n\
 ");
 printf("#endif\n\n");
 
