diff -Nru zgv-5.7/config.mk zgv-5.7.new/config.mk
--- zgv-5.7/config.mk	Sat Nov  4 15:55:35 2000
+++ zgv-5.7.new/config.mk	Sat Nov  4 15:54:39 2000
@@ -12,7 +12,7 @@
 # This is likely to be what you'll want for most systems:
 #
 CC=gcc
-CFLAGS=-O2 -Wall -fomit-frame-pointer -finline-functions
+CFLAGS=$(OPTFLAGS) -Wall
 #
 # If you're brave enough to try compiling zgv on a non-x86 system :-),
 # this might be a better bet:
@@ -63,7 +63,7 @@
 #
 # If you don't know what to do, leave it as-is.
 #
-#INFO_DIR_UPDATE=no
+INFO_DIR_UPDATE=no
 
 
 # ------------------- Format-related options ----------------------
--- zgv-5.4/doc/Makefile.orig	Thu Dec 21 04:31:28 2000
+++ zgv-5.4/doc/Makefile	Mon Jul 16 02:42:05 2001
@@ -34,8 +34,9 @@
 else
 install: ../src/install-info
 endif
-	install -m 644 zgv.1 $(MANDIR)
-	install -m 644 zgv zgv-? $(INFODIR)
+	install -d $(DESTDIR)$(MANDIR) $(DESTDIR)$(INFODIR)
+	install -m 644 zgv.1 $(DESTDIR)$(MANDIR)
+	install -m 644 zgv zgv-? $(DESTDIR)$(INFODIR)
 # Update info `dir' file.
 # Info always uses a dir file in preference to a dir.gz, so we don't use
 # dir.gz unless it's the only game in town.
--- zgv-5.7/src/Makefile.orig	2003-04-05 21:50:50.000000000 +0200
+++ zgv-5.7/src/Makefile	2003-08-02 20:50:14.000000000 +0200
@@ -51,12 +51,12 @@
 
 install: all
 	@if [ "$(BACKEND)" = SVGALIB ]; then \
-	  echo install -m 4755 -o root -g root -s zgv $(BINDIR); \
-	  install -m 4755 -o root -g root -s zgv $(BINDIR); \
+	  echo install -D -m 755 zgv $(DESTDIR)$(BINDIR)/zgv; \
+	  install -D -m 755 zgv $(DESTDIR)$(BINDIR)/zgv; \
 	fi
 	@if [ "$(BACKEND)" = SDL ]; then \
-	  echo install -m 755 -s zgv $(BINDIR)/zgv-sdl; \
-	  install -m 755 -s zgv $(BINDIR)/zgv-sdl; \
+	  echo install -D -m 755 zgv $(DESTDIR)$(BINDIR)/zgv-sdl; \
+	  install -D -m 755 zgv $(DESTDIR)$(BINDIR)/zgv-sdl; \
 	fi
 
 uninstall:
