--- zgv-5.9/src/zgv_io.c~	2005-01-21 00:07:46.000000000 +0100
+++ zgv-5.9/src/zgv_io.c	2007-07-03 18:53:32.425024420 +0200
@@ -646,7 +646,7 @@
         case SDLK_DELETE:	return(RK_DELETE);
         case SDLK_RETURN:	return(RK_ENTER);
         default:
-          /* stop complaints */
+          /* stop complaints */;
         }
 
       if(event.key.keysym.sym==SDLK_SPACE && (event.key.keysym.mod&KMOD_CTRL))
--- zgv-5.9/src/modesel.c~	2003-04-05 16:41:04.000000000 +0200
+++ zgv-5.9/src/modesel.c	2025-09-26 20:49:05.303326233 +0200
@@ -6,6 +6,7 @@
  */
 
 #include <stdio.h>
+#include <stdlib.h>
 #include "zgv_io.h"
 #include "readnbkey.h"
 #include "modesel.h"
